uwblueprint/humane-society
# Create login frontend page

## Notion ticket link

[Login Page](https://www.notion.so/uwblueprintexecs/Login-Page-25e845c93c0941fc8ab00bcb83653c42?pvs=4)



## Implementation description
* Modified the Login to now display an email and password field, with validation
* Uses Redirect on success to go to pets
* If viewed on a phone in landscape, it becomes scrollable


## Steps to test
1. Go to localhost:3000/login, make sure data is cleared
2. Both fields should throw error if invalid/empty field
3. On success, should redirect to /pets


## What should reviewers focus on?
* Logging in and redirectly correctly
* Throwing errors correctly
* Any coding style/visual things that look off
* Unlimited attempts at logging in?

## Checklist
- [ x] My PR name is descriptive and in imperative tense
- [ x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ x] I have run the appropriate linter(s)
- [ x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR


## Scrrenshots
<img width="1440" alt="Screenshot 2024-11-27 at 12 21 10 AM" src="https://github.com/user-attachments/assets/7e334e42-f26f-41c8-923e-f421df80f9d1">
<img width="1440" alt="Screenshot 2024-11-27 at 12 21 31 AM" src="https://github.com/user-attachments/assets/9384deef-2737-4540-bb11-fee8fa7cd032">
<img width="1440" alt="Screenshot 2024-11-27 at 12 21 52 AM" src="https://github.com/user-attachments/assets/7ae9700b-a260-49e6-b30b-99cf3d8f86c2">
<img width="582" alt="Screenshot 2024-11-27 at 12 22 18 AM" src="https://github.com/user-attachments/assets/bd53d62c-08e4-4188-b4ee-288a973bb7d9">
